examples/plot_enet_use_case.py

```diff
@@ -0,0 +1,157 @@
+"""
+===============================================================
+ElasticNet vs. Lasso: Illustration of ElasticNet use case
+===============================================================
+
+The Lasso estimator is useful in features selection thanks
+to its ability to produce sparse solutions. However,
+the latter becomes ineffective when it comes to high dimensional
+datasets as the support of the produced solution is limited
+by the number of samples.
+
+The ElasticNet estimator extend the feature selection property
+of Lasso estimator by producing sparse solutions regardless
+of the number of samples.
+
+Below is an example that illustrates that.
+"""
+import warnings
+
+from celer import ElasticNetCV, LassoCV
+from sklearn.model_selection import KFold
```

Comment:
no need for CV here IMO. Just pick alpha a la mano and try to get convincing different solutions